### PR TITLE
Generate npm test

### LINKED ISSUE #5: Generate test folder
Like in [generator-jhipster-fortune](https://github.com/jdubois/generator-jhipster-fortune), this generator should generate :
- a test folder
- with app.js (or something else)
- maybe gulpfile.js too
- the generated package.json must be update too

So the users will have to write tests in their module

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Generate test folder in generated project
Fix #5

### CODE REVIEW

File: generators/app/templates/_package.json [hunk 1]
```diff
@@ -28,19 +28,13 @@
     "generator-jhipster": ">2.26.0"
   },
   "devDependencies": {
-    "gulp": "^3.9.0",
```
Comment (deepu105) [inline generators/app/templates/_package.json]: Why did you remove the gulp dependencies?

Comment (pascalgrimaud) [inline generators/app/templates/_package.json]: I didn't see any use.
Maybe I miss something?

Comment (deepu105) [inline generators/app/templates/_package.json]: Its used by the gulpfile generated on the project root while running this
generator.

The gulp file has tasks to automate publishing of module so that module
author can just do gulp deploy-major|minor|patch to do version bump based
on the major|minor|patch, git commit.git push, tag, release and npm publish
in one step instead of doing these manually checkout the gulp file, its
same as the gulpfile I added on this generator root as well, you can try it
out for your next patch release may be just do gulp deploy-patch

Thanks & Regards,
Deepu

Comment (pascalgrimaud) [inline generators/app/templates/_package.json]: Oh you're right. Sorry, I didn't have a closed look to your last PR and the new file .gulpfile.js !
I will fix that!
